src/stories/containers/finances-overview/Components/YearPicker/YearPicker.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import YearPicker from './YearPicker';
+import type { ComponentMeta } from '@storybook/react';
+import type { FigmaParams } from 'storybook-addon-figma-comparator/dist/ts/types';
+
+export default {
+  title: 'Components/Finances/YearPicker',
+  component: YearPicker,
+  chromatic: {
+    viewports: [375, 834],
+    pauseAnimationAtEnd: true,
+  },
+} as ComponentMeta<typeof YearPicker>;
+
+const variantsArgs = [
+  {
+    years: [2021, 2022, 2023],
+    yearSelect: 2022,
```

Comment:
You changed this args so you should update this as well

Reply:
done